libs/components/datetime/src/lib/modules/datepicker/datepicker-calendar-inner.component.html

```diff
@@ -21,9 +22,8 @@
             'sky-btn-disabled': datepickerMode === maxMode
           }"
           (click)="toggleModeCalendar($event)"
+          >{{ title }}</button
```

Comment:
Putting this here as the html file isn't in the PR.

I noticed that the days of the week in the daypicker are still using a `b` tag to be bolded instead of the token specified in the design spec.